man/docker-run.1.md

```diff
@@ -508,6 +509,18 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Docker to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+To control mount propagation property of volume one can use `:[r]shared`,
+`:[r]slave` or `:[r]private` propagation flag. Propagation property can
+be specified only for bind mounted volumes and not for internal volumes or
+volume driver volumes. For this to work source mount point (mount point where
+source dir is mounted on) has to have right propagation properties. For shared
+volumes, source mount point has to be shared. And for slave volumes, source
+mount has to be either shared or slave. Use `df <source-dir>` to figure out
+the source mount and then use `findmnt -o TARGET,PROPAGATION <source-mount-dir>`
```

Comment:
One can also parse `/proc/self/mountinfo` and go through the mount entry of source mount. One needs to look at field 7 (optional fields) and see if there are any propagation properties specified for mount.

Following is from `man proc`.

`shared:X          mount is shared in peer group X
master:X          mount is slave to peer group X`

If nothing is specified, then mount point is private.

`findmnt` comes from util-linux package. I think it would be good if util-linux utilities are available in boot2docker. I have never used boot2dokcer. You will have to help me figure out how to make findmnt available on boot2docker. Otherwise we will have to resort to just explain looking at /proc/self/mountinfo which is not very easy.